shop/src/main/java/ro/msg/learning/shop/service/OrdersService.java

```diff
@@ -0,0 +1,16 @@
+package ro.msg.learning.shop.service;
+
+import ro.msg.learning.shop.entity.Orders;
+
+import java.util.Collection;
+import java.util.UUID;
+
+public interface OrdersService {
+
+    public abstract void createOrders(Orders orders);
```

Comment:
you don't need the abstract keyword as this is an interface. (same for all the methods in Service interfaces). Also why void?